test/controllers/site_controller_test.rb

```diff
@@ -1,7 +1,22 @@
 require 'test_helper'
 
 class SiteControllerTest < ActionController::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  
+   
+  test "should get index" do
+    get :index
+
+    assert_template :index
+    assert_select "title", "Home | RubyGems AdoptionCenter"
+    assert_response :success
+  end
+
+  test "should get search" do
+    get :search_gems, search: 'rails'
+
+    assert_template :search_gems
+    assert_select "title", "| RubyGems AdoptionCenter"
+    assert_response :success
+  end
```

Comment:
@linatodi the idea of having test and doing TDD is to have test allow us to explain explain what the expected behaviour of the software is. I think this test is not really testing that.

We should probably be testing that the gem we searched for actually appears in the results and also that if we search for a gem that doesn't exist the application can gracefully handle that type of situation.